simulator/CARLA_Autoware/carla_pointcloud_interface/src/carla_pointcloud_interface/carla_pointcloud_interface_node.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright 2024 Raditya.
```

Comment:
Please update all copyrights as follows:
```suggestion
// Copyright 2024 Autoware Foundation. All rights reserved.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
```